cristin-organization/src/main/java/no/unit/nva/cristin/organization/HttpExecutorImpl.java

```diff
@@ -52,11 +67,47 @@ public HttpExecutorImpl(HttpClient client) {
         this.httpClient = client;
     }
 
+    public Organization getOrganization(URI uri) throws ApiGatewayException {
+        return fromSubSubunit(getSubSubUnitDtoWithMultipleEfforts(uri));
+    }
+
+    protected SearchResponse<Organization> query(URI uri) throws NotFoundException, FailedHttpRequestException {
+        HttpResponse<String> response = sendRequestMultipleTimes(addLanguage(uri)).get();
+        if (isSuccessful(response.statusCode())) {
+            try {
+                List<Organization> organizations = getOrganizations(response);
+                return new SearchResponse<Organization>(uri)
+                        .withHits(organizations)
+                        .withSize(getCount(response, organizations));
+            } catch (JsonProcessingException e) {
+                throw new FailedHttpRequestException(e.getMessage());
```

Comment:
Is json exception returned to client? Should we return something else?

Reply:
We go for json-ld as the default

Reply:
What I mean is: I see the statuscode is Bad Gateway, but what about the message back to client? Should we return the content of a JsonProcessingException back to the client, or should we hide the internal error and return a more generic one to the client?

Reply:
I read something else in the first place...  We should hide/camouflage the json exception as something more infomative